types/readable-stream/index.d.ts

```diff
@@ -409,15 +409,15 @@ declare namespace _Readable {
         destroy?(this: _IReadable, error: Error | null, callback: (error: Error | null) => void): void;
     };
 
-    class Readable extends _Readable {
+    class Readable extends _Readable implements NodeJS.ReadableStream {
         readonly readableAborted: never;
         readonly readableDidRead: never;
         readonly readableEncoding: never;
         readonly readableEnded: never;
         readonly readableObjectMode: never;
 
         constructor(options?: ReadableOptions);
-        pipe<T extends _IWritable>(destination: T, options?: { end?: boolean | undefined }): T;
+        pipe<T extends NodeJS.WritableStream>(destination: T, options?: { end?: boolean | undefined }): T;
```

Comment:
Looks correct to me, and unlikely to be breaky: `NodeJS.WritableStream` is a supertype of `_IWritable`, so this now accepts all the `destination`s it used to, plus some.